Explorer/Assets/Scripts/CrdtEcsBridge/ComponentWriter/IECSToCRDTWriter.cs

```diff
@@ -0,0 +1,16 @@
+using CRDT;
+using CrdtEcsBridge.Serialization;
+
+namespace CrdtEcsBridge.ECSToCRDTWriter
+{
+    public interface IECSToCRDTWriter
+    {
+        void PutMessage<T>(CRDTEntity crdtID, int componentId, T model);
```

Comment:
You should constrain `T` to `IMessage<T>` as we are using `Protobuf` messages for communication